src/providers/authProvider/index.ts

```diff
@@ -37,18 +35,24 @@ const setToken = (token: string): void => {
   // date.setTime(date.getTime() + 24 * 60 * 60 * 1000)
   // const expires = date.toUTCString()
   // document.cookie = `${constants.TOKEN_KEY}=${token}; expires=${expires}; path=/ `
-  localStorage.setItem('AccessToken', token)
+  localStorage.setItem('accessToken', token)
 }
 
 export const removeUserToken = (): void => {
-  removeCookie(constants.TOKEN_KEY)
+  removeCookie()
+  removeLocalStorageItem('accessToken')
 }
 
-const removeCookie = (name: string): void => {
-  // note: setting the expiry date of a cookie to a past data effectively
-  // removes it
-  console.log(name)
-  document.cookie = `${name}=; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/;`
+const removeCookie = (): void => {
+  document.cookie =
+    'accessToken=; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/;'
+
+  document.cookie =
+    'refreshToken=; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/;'
+}
```

Comment:
is there a reason why we set the cookie twice?  If there is, could you add a comment please?